poweremail_signaturit/poweremail_core.py

```diff
@@ -119,5 +119,20 @@ def parse_body_html(pem_body_html, pem_body_text):
             res = response.get('error_message', response['error'])
         return res
 
+    def get_mail_audit_trail(self, cr, uid, ids, signature_id, certificate_id, context=None):
+        if context is None:
+            context = {}
+
+        client = get_signaturit_client()
+        pdf = client.download_email_audit_trail(signature_id, certificate_id)
+        return pdf
+
+    def get_email(self, cr, uid, ids, signature_id, context=None):
```

Comment:
```suggestion
    def get_signaturit_email(self, cr, uid, ids, signature_id, context=None):
```
el nom de get_email es poder massa generic i pot comportar confusió